tool/tctl/common/plugins_command.go

```diff
@@ -45,10 +59,27 @@ func (p *PluginsCommand) Initialize(app *kingpin.Application, config *servicecfg
 	p.dryRun = true
 
 	pluginsCommand := app.Command("plugins", "Manage Teleport plugins.").Hidden()
+
 	p.cleanupCmd = pluginsCommand.Command("cleanup", "Cleans up the given plugin type.")
 	p.cleanupCmd.Arg("type", "The type of plugin to cleanup. Only supports okta at present.").Required().EnumVar(&p.pluginType, string(types.PluginTypeOkta))
 	p.cleanupCmd.Flag("dry-run", "Dry run the cleanup command. Dry run defaults to on.").Default("true").BoolVar(&p.dryRun)
 
+	p.install.cmd = pluginsCommand.Command("install", "Install new plugin instance")
+
+	p.install.scim.cmd = p.install.cmd.Command("scim", "Install a new SCIM integration")
+	p.install.scim.cmd.
+		Flag("name", "The name of the SCIM plugin resource to create").
+		Default("scim").
+		StringVar(&p.install.name)
+	p.install.scim.cmd.
+		Flag("saml-connector", "The name of the Teleport SAML connector users wil log in with.").
+		Required().
```

Comment:
So for our specific use-case it's probably fine for now, but what if you're not using any SSO connector, or something other than Okta SSO? Will this feature still work, can connector be optional?

And if it's not optional, do we validate it somehow?

Reply:
Added validation